.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/cakephp) for that.
```

Comment:
While I like that, why not make it a real template? Like so:
* * *

This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)
... issue.

After doing some research I am sure that this is not a question or help ticket. I did check the book, API, irc channel and stackoverflow, first.

I did also double check if a similiar issue already exists and/or has been closed before.

## DESCRIPTIVE TITLE HERE
CakePHP Version: FULL SEM-VER OR COMMIT ID, HERE.
Platform and Target: YOUR WEB-SERVER, DATABASE AND OTHER RELEVANT INFO AND HOW THE REQUEST IS BEING MADE, HERE.

### Expected Behavior
EXPLAIN WHAT IS TO BE EXPECTED, HERE.

### Actual Behavior
EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.

### Relevance
EXPLAIN WHY YOU THINK STUFF SHOULD BE CHANGED, HERE.

### Isolation (BUG ONLY)
IF A BUG REPORT, SPECIFY HOW TO ISOLATE THE BUG HERE AND/OR PROVIDE A UNIT TEST CASE, HERE.

### Solution
OPTIONAL: YOUR SUGGESTION ON HOW TO SOLVE THE ISSUE, HERE. IF BREAKING BACKWARDS-COMPATIBILITY OR CHANGING CONVENTIONS, EXPLAIN WHY, HERE.